dom-manipulation/main.js

```diff
@@ -0,0 +1,24 @@
+var $hotButton = document.querySelector('.hot-button');
+// eslint-disable-next-line no-unused-vars
+var $clickCount = document.querySelector('.click-count');
+var clicksHit = 1;
+
+function clicks(event) {
+  $clickCount++;
+  document.querySelector('.click-count').textContent = 'Clicks: ' + clicksHit++;
```

Comment:
Once you fix the above problem, you should use `$clickCount` rather than querying again